prov/efa/src/efa.h

```diff
@@ -383,6 +386,17 @@ bool efa_ep_support_rdma_read(struct fid_ep *ep_fid)
 	return efa_ep->domain->ctx->device_caps & EFADV_DEVICE_ATTR_CAPS_RDMA_READ;
 }
 
+static inline
+bool efa_ep_support_rnr_retry_modify(struct fid_ep *ep_fid)
+{
+	struct efa_ep *efa_ep;
+#ifndef HAVE_CAPS_RNR_RETRY
+	return false;
+#endif
+	efa_ep = container_of(ep_fid, struct efa_ep, util_ep.ep_fid);
+	return efa_ep->domain->ctx->device_caps & EFADV_DEVICE_ATTR_CAPS_RNR_RETRY;
```

Comment:
I don't think this will compile for rdma-core versions without `EFADV_DEVICE_ATTR_CAPS_RNR_RETRY`? You need an else block.